MarsMission/Views/SearchViewCell.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  SearchViewCell.swift
+//  MarsMission
+//
+//  Created by Thanyani on 2020/11/05.
+//
+
+import UIKit
+
+class SearchViewCell: UICollectionViewCell {
+	
+	private lazy  var viewModel =  WeatherDataViewModel(view: self)
```

Comment:
Why is the view model in the cell and not the view controller

Reply:
<img width="738" alt="Screenshot 2020-11-06 at 16 21 25" src="https://user-images.githubusercontent.com/55865505/98376487-401a4980-204c-11eb-936e-9fe07ae6e5bb.png">